storage/innobase/log/log0crypt.cc

```diff
@@ -441,7 +441,7 @@ bool log_tmp_block_encrypt(
 	uint64_t	offs,
 	bool		encrypt)
 {
-	uint dst_len;
+	uint dst_len = (uint) size;
```

Comment:
Please do not use a C-style cast in C++ code. You have used `static_cast<uint>(size)` elsewhere.